singularity-prefix.def

```diff
@@ -0,0 +1,9 @@
+Bootstrap: docker
+From: centos:8.2.2004
+
+%post
+    dnf install -y gcc gcc-c++ diffutils
+
+%environment
+    export LC_ALL=C
+    export PATH=/usr/games:$PATH
```

Comment:
```suggestion
    dnf install -y gcc gcc-c++ diffutils wget
    
     # Add Gentoo Prefix Bootstrap
    wget https://gitweb.gentoo.org/repo/proj/prefix.git/plain/scripts/bootstrap-prefix.sh -P /usr/local/bin
    chmod +x /usr/local/bin/bootstrap-prefix.sh

%environment
    export LC_ALL=C
    export PATH=/usr/games:/usr/local/bin:$PATH
```

Reply:
For now this will probably work, but I think it's better to include our own version/copy of the bootstrap script. That's a version that we tested (and possibly we want to modify it a bit). Always taking the latest version from the Gentoo git might give unexpected errors/results.

Reply:
Jup, agreed. And we've seen that the bootstrap script isn't terribly stable...